client/coral-admin/src/reducers/community.js

```diff
@@ -45,6 +46,14 @@ export default function community (state = initialState, action) {
     commenters[idx].roles[0] = action.role;
     return state.set('commenters', commenters.map(id => id));
   }
+  case SET_COMMENTER_STATUS: {
+    const commenters = state.get('commenters');
+    const idx = commenters.findIndex(el => el.id === action.id);
+
+    commenters[idx].status = action.status;
```

Comment:
I this code is modifying a immutable object, not a js object. So I think it should be:

`const commenter = commenters[idx].set('status', action.status);`

and then you have to use maybe `commenters.setIn(idx, commenter)` to update the array

@impronunciable thoughts?